src/integrationTest/java/uk/gov/hmcts/darts/arm/service/DetsToArmBatchProcessResponseFilesIntTest.java

```diff
@@ -46,11 +41,7 @@ class DetsToArmBatchProcessResponseFilesIntTest extends AbstractArmBatchProcessR
     private DetsDataManagementConfiguration detsDataManagementConfiguration;
     @Autowired
     private ObjectStateRecordRepository osrRepository;
-    @Autowired
-    private ExternalDetsDataStoreDeleter externalDetsDataStoreDeleter;
-    @MockBean
-    private DetsDataStoreDeleter detsDataStoreDeleter;
```

Comment:
`DetsDataStoreDeleter` has no usages other than tests and the now redundant `ExternalDetsDataStoreDeleter`, so can it be removed also, or will it see usage in the future async version of the flow?

Reply:
I left this as it will be needed for the async flow, where the other stuff will likely get moved to a new service :)